sdk/jobs/automl-standalone-jobs/automl-classification-task-bankmarketing/automl-classification-task-bankmarketing.ipynb

```diff
@@ -323,13 +323,16 @@
   }
  ],
  "metadata": {
+  "interpreter": {
+   "hash": "69daf304a5afb4d75ee1bae08a753430bc753de517d51cf450bbf47abb30c09c"
+  },
```

Comment:
are these expected change?

Reply:
I did run the jupyter clear-output command on these files, let me remove this manually (don't think it has any effect though, it'll probably be updated with another interpreter hash when the notebook is attached to a kernel)